README.md

```diff
@@ -31,37 +31,43 @@ Colleagues may collaborate on a spreadsheet that looks like this:
 | ...          |                   |                                       |
 
 You can export your spreadsheet to CSV, run it through the ECS mapper,
-and get your starter pipelines generated.
+and generate your starter pipelines.
 
-Note that this tool generates starter pipelines. They only do the rename/copy
-operations and some field format adjustments. It's up to you to integrate them
+Note that this tool generates starter pipelines. They only do field rename and copy
+operations as well as some field format adjustments. It's up to you to integrate them
 in a complete pipeline that ingests and outputs the data however you need.
 
-Scroll down to the [Examples](#examples) section below, to get right to a
+Scroll down to the [Examples](#examples) section below to get right to a
 concrete example you can play with.
 
 ## Maturity
 
 This code is a proof of concept and is not officially supported.
-The pipelines generated by this tool are not meant to be complete, nor production-ready.
+The pipelines generated by this tool are likely not complete and probably need more
+testing and validation before they are ready for production.
 They are simply meant to give you a head start in mapping various sources to ECS.
 
 ## CSV Format
 
-Here's more details on the CSV format supported by this tool. Since mapping
-spreadsheets are primarily used by humans, it's totally fine to have as many columns
+Here are more details on the CSV format supported by this tool. Since mapping
+spreadsheets are used by humans, it's totally fine to have as many columns
 as you need in your spreadsheets/CSV. Only the following columns will be considered:
 
 | column name | required | allowed values | notes |
 |-------------|----------|----------------|-------|
 | source\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Lines with empty "source\_field" are skipped. |
 | destination\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Can be left empty if there's no copy action (just a type conversion). |
-| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
+| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, parse\_timestamp, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
+| timestamp\_format | optional | UNIX, UNIX\_MS, ISO8601, TAI64N, or a Java time pattern. |
```

Comment:
I'd like this note to set expectations clearly.

We've already determined that both UNIX shorthands are supported by all 3 👍 

1. Are you positive that ISO8601 and TAI64N are also supported by all 3?
2. The "Java time pattern" will not work in Beats.

I think we should clearly state that if folks need a specific shorthand or explicit date pattern, ecs-mapper will output it as is in the resulting pipelines, without validation. This enables users to do exactly what they want once they know which type of pipeline they want to do: they can use exactly what their chosen tech supports. But at this point the other 2 types of pipelines would no longer necessarily be valid.

Now depending on your answer to 1, maybe this applies to these two shorthands as well. But it definitely applies to the Java time patterns.

So perhaps in the table itself we can say something generic like "or pipeline-specific shorthands or time formats" with an asterisk, and later we expand the thinking explained just above?